test/templates/list-group/index.njk

```diff
@@ -1,31 +1,37 @@
 {% extends "layout.njk" %}
 {% from "./component.njk" import list_group %}
-{% from "../form/component.njk" import form_item %}
-{% from "../button/component.njk" import button %}
-{% from "../icon/component.njk" import icon %}
 {% from "./../format.njk" import format %}
 
 {% block content %}
+<style>
+
+.test-pg__checkbox{
```

Comment:
This looks great! 

@greg-a-smith just merged a pr with a shared-layout stylesheet for the placeholder styles, can we move this in there?

 https://github.com/SAP/fundamental-styles/blob/master/test/resources/layout-shared.css

Reply:
sure!